salt/base/scripts/sentry-checkin.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+MINION_ID=$(salt-call --local grains.get id --out=newline_values_only)
```

Comment:
Should all of these be templated verbatim or do we need to do `salt-call` each time? Seems we could just use the value from pillar/pillar_ext when we write/update the file.

Reply:
[130b807](https://github.com/python/psf-salt/pull/585/commits/130b8073359ae81259a615fb46399115b3d57d13)